sys/net/gnrc/netapi/gnrc_netapi.c

```diff
@@ -144,29 +133,3 @@ int gnrc_netapi_dispatch(gnrc_nettype_t type, uint32_t demux_ctx,
 
     return numof;
 }
-
-int gnrc_netapi_send(kernel_pid_t pid, gnrc_pktsnip_t *pkt)
```

Comment:
If you keep the implementation here but make these short functions inline, you can leave `_send_recv` and `_get_set` hidden in here while still gaining the size optimisation?!

Reply:
as @gebart was pointing out.

Reply:
I don't know what you are referring to here. @gebart just pointed out that the code optimization comes from inlining those two instead of `_gnrc_netapi_send_recv()` which you said is a bad thing, because it exposes this functions.